Mathlib/Topology/Semicontinuous.lean

```diff
@@ -360,6 +363,15 @@ theorem Continuous.comp_lowerSemicontinuous_antitone {g : γ → δ} {f : α →
   hg.continuousAt.comp_lowerSemicontinuousAt_antitone (hf x) gmon
 #align continuous.comp_lower_semicontinuous_antitone Continuous.comp_lowerSemicontinuous_antitone
 
+theorem LowerSemicontinuousAt.comp {f : α → β} {g : ι → α} {x : ι}
+    (hf : LowerSemicontinuousAt f (g x)) (hg : ContinuousAt g x) :
```

Comment:
Could you instead write it as
```suggestion
    (hf : LowerSemicontinuousAt f y) (hg : ContinuousAt g x) (h : y = g x) :
```
This form is often much more convenient in practice. It's not the form we had adopted early in the library, but with hindsight it's really better, and I should probably convert other similar theorems to this form.